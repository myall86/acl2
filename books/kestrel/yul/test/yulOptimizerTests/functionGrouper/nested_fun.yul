{
    let a
    function f() {
        let b
        function g() {
            let c
        }
        let d
    }
}
// ====
// dialect: yul
// ----
// step: functionGrouper
//
// {
//     { let a }
//     function f()
//     {
//         let b
//         function g()
//         { let c }
//         let d
//     }
// }
