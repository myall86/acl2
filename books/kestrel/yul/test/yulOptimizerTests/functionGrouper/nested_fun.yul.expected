
 {
     { let a }
     function f()
     {
         let b
         function g()
         { let c }
         let d
     }
 }
